README.md

```diff
@@ -231,9 +231,41 @@ If you have an asset file's data loaded into memory, you can sign the the asset
 import { readFile } from 'node:fs/promises';
 import { createC2pa, createTestSigner } from 'c2pa-node';
 
+// read an asset into a buffer
+const buffer = await readFile('to-be-signed.jpg');
+const asset: Asset = { buffer, mimeType: 'image/jpeg' };
+
+// build a manifest to use for signing
+const manifest = new ManifestBuilder(
+  {
+    claim_generator: 'my-app/1.0.0',
+    format: 'image/jpeg',
+    title: 'buffer_signer.jpg',
+    assertions: [
+      {
+        label: 'c2pa.actions',
+        data: {
+          actions: [
+            {
+              action: 'c2pa.created',
+            },
+          ],
+        },
+      },
+      {
+        label: 'com.custom.my-assertion',
+        data: {
+          description: 'My custom test assertion',
+          version: '1.0.0',
+        },
+      },
+    ],
+    },
+  { vendor: 'cai' },
+);
+
+// create a signing function
 async function sign(asset, manifest) {
```

Comment:
asset/buffer moved out since they'll be passed in as params to the sign function